doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md

```diff
@@ -5,4 +5,102 @@ permalink: /:path/
 sort_idx: 20
 ---
 
-SDK integration instructions for select platforms
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+# The guide provides information on how to cross-compile C++ SDK for Raspberry PI
+
+**Verified against:**
+
+**Host OS:** Ubuntu 14.04 LTS Desktop 64-bit.
+
+**The further instructions must be executed on the host machine.**
+
+1. Download and install toolchain.
+
+        mkdir rpi_root && cd rpi_root
+        export RPI_ROOT=$(pwd)
+        git clone https://github.com/raspberrypi/tools.git
+        export INSTALLDIR=$RPI_ROOT/tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian
+        export PATH=$PATH:$INSTALLDIR/bin
+        export CROSS=arm-linux-gnueabihf
+
+2. Download and install zlib.
+
+        cd $RPI_ROOT
+        wget http://zlib.net/zlib-1.2.8.tar.gz
+        tar -xvzf zlib-1.2.8.tar.gz
+        cd zlib-1.2.8
+        CROSS_PREFIX="${CROSS}-" prefix=${INSTALLDIR}/arm-linux-gnueabihf/libc/usr ./configure
```

Comment:
This pattern is given in the configure script as an example. Also `CROSS_PREFIX` AFAIK should be set before the `./configure` because it is an environment variable. So the command is going to be `CROSS_PREFIX="${CROSS}-"  ./configure prefix=${INSTALLDIR}/arm-linux-gnueabihf/libc/usr`. I consider this form seems a little strange, what do you think about it?